<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeCalibrate</class>
 <widget class="QWidget" name="TimeCalibrate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lbTitle">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>时间校准</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbYear">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>年</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbMonth">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>190</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>月</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbDay">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>190</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>日</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbHour">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>时</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbMinute">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>270</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>分</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbSecond">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>270</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>秒</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbDateTime">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>时间日期：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leYear">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>71</width>
     <height>40</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="leMonth">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>71</width>
     <height>40</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="leDay">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>200</y>
     <width>71</width>
     <height>40</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="leHour">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>71</width>
     <height>40</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="leMinute">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>280</y>
     <width>71</width>
     <height>40</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="leSecond">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>280</y>
     <width>71</width>
     <height>40</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pbExit">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>390</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>641</width>
     <height>171</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pbCalibrate">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>60</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>校准</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lbSysDateTime">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>361</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>lbSysDateTime</zorder>
  <zorder>frame</zorder>
  <zorder>lbTitle</zorder>
  <zorder>lbYear</zorder>
  <zorder>lbMonth</zorder>
  <zorder>lbDay</zorder>
  <zorder>lbHour</zorder>
  <zorder>lbMinute</zorder>
  <zorder>lbSecond</zorder>
  <zorder>lbDateTime</zorder>
  <zorder>leYear</zorder>
  <zorder>leMonth</zorder>
  <zorder>leDay</zorder>
  <zorder>leHour</zorder>
  <zorder>leMinute</zorder>
  <zorder>leSecond</zorder>
  <zorder>pbExit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
